packages/ember-toucan-core/src/components/form/fields/autocomplete.ts

```diff
@@ -42,7 +42,7 @@ export interface ToucanFormAutocompleteFieldComponentSignature {
     label?: string;
 
     /**
-     * A string to display when there are no results after the user's filter.
+     * A string to display when there are no results after filtering.
      */
     noResultsText?: string;
```

Comment:
Jinx! haha yeah.  I'm thinking maybe we make it required here to force folks to provide us with (hopefully!) translated strings for what should go there.  Without it being required, it could also be a little weird and lead to inconsistent UX if someone forgets to provide it 🙈

Reply:
Nice. I'll take it as a followup.